osu.Game/Migrations/OsuDbContextModelSnapshot.cs

```diff
@@ -141,6 +141,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("VideoFile");
 
+                    b.Property<int>("VideoOffset");
```

Comment:
I've already removed both `VideoOffset` and `VideoFile` from the actual `BeatmapMetadata`, should I put `VideoFile` back?

Reply:
That is *not* what I'm saying. The column spec should not be in the db context snapshot if the migration is gone and so are the fields (which I intended to highlight by leaving a comment at this exact line).

Reply:
Sorry, hadn't realized that I left `VideoFile` in the metadata.
When creating a migration for removing `VideoFile`, it failed because dropping columns is not supported. How should it be dealt with?

Reply:
Huh, you're right, [sqlite doesn't support it](https://docs.microsoft.com/en-us/ef/core/providers/sqlite/limitations#migrations-limitations). Wow.

In that case I'm not too sure? We could rebuild the table altogether but that's going to require hand-writing SQL, most likely. Maybe better to just leave it unused [as we were planning to ditch EF Core and sqlite in the long run](https://github.com/ppy/osu/issues/7057)?